server/test/integration/vitalsTest/handleHeartbeatTest.js

```diff
@@ -215,6 +215,17 @@ describe('vitals.js integration tests: handleHeartbeat', () => {
       sandbox.restore()
     })
 
+    it('should log a Sentry alert when the INS sensor value is less or equal to zero', async () => {
```

Comment:
Quick note: because we need to deploy this PR very soon, maybe you can add a follow-up PR for the test cases in this file: you should add `isINSZero: false` to every other `data` request in this file, to make sure that that field does not change the logic of the other test cases.